clients/web/src/models/CollectionCreationPolicyModel.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'underscore';
+import AccessControlledModel from 'girder/models/AccessControlledModel';
+import { restRequest } from 'girder/rest';
+
+var CollectionCreationPolicyModel = AccessControlledModel.extend({
+    resourceName: 'system/setting/collection_creation_policy',
+    fetchAccess: function () {
+        return restRequest({
+            path: this.resourceName + '/access',
+            type: 'GET'
+        }).done(_.bind(function (resp) {
+            this.set('access', resp);
+            this.trigger('g:accessFetched');
+            return resp;
+        }, this)).error(_.bind(function (err) {
```

Comment:
Makes sense. We didn't replace all the existing occurrences of it, however it's good to clean them up when you encounter them, and not add any new ones unless they're necessary. 😄